src/transformer/schema-transformer.ts

```diff
@@ -153,42 +153,61 @@ export class SchemaTransformer {
         // Example: Mutation.createChannel.response
         let args = file.split('.');
         let typeName: string = args[0];
-        let name: string = args[1];
+        let fieldName: string = args[1];
         let templateType = args[2]; // request or response
+
+        // default to composite key of typeName and fieldName, however if it
+        // is Query, Mutation or Subscription (top level) the compositeKey is the
+        // same as fieldName only
+        let compositeKey = `${typeName}${fieldName}`;
+        if (statements.indexOf(typeName) >= 0) {
```

Comment:
Can you remove `Subscription` from line 148 statements array? This will align the subscriptions with the none data source composite keys